src/dmd/dtemplate.d

```diff
@@ -6179,14 +6179,12 @@ extern (C++) class TemplateInstance : ScopeDsymbol
      * Compare proposed template instantiation with existing template instantiation.
      * Note that this is not commutative because of the auto ref check.
      * Params:
-     *  o = existing template instantiation
+     *  ti = existing template instantiation
      * Returns:
-     *  0 for match, 1 for no match
+     *  true for match
      */
-    override final int compare(RootObject o)
+    final bool equalsx(TemplateInstance ti)
```

Comment:
You repeated this twice so I'm assuming thats not a typo and you did mean to put the x on the end?

Reply:
Yes, I wanted to clearly distinguish it from the one in `RootObject`.